app/Exceptions/ModuleExistsException.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Exceptions;
+
+class ModuleExistsException extends AbstractHttpException
+{
+    private $module;
+
+    public function __construct($module)
```

Comment:
Rename this to `$module_name`, makes it clearer that we're not passing in a module instance